guides/_posts/2014-01-11-ifelse.html

```diff
@@ -36,73 +49,83 @@
 
 {% endhighlight %}
 
-But, any property can be used within an `@if`, `@else` conditional.  So, if you want a layout to be responsive to a panel's width:
+But, any property can be used within an `@if`, `@else` conditional.  So, if you want a layout to be responsive to a section's width:
 
 {% highlight css %}
 
-/* GSS: */
-@if #panel[width] >= 800 {
-  /* ... */
-}
-@else #panel[width] >= 500 {
-  /* ... */
+@if section[width] <= 400 {
+  section {
+    article {
+      :first[top] == ^[top];
+      bottom: == :next[top];
+      left: == ^[left];
+      width: == ^[width];
+      height: == ^[height] / 2;
+    }
+  }
 }
-@else {
-  /* ... */
+@else  {
+  section {
+    article {
+      width: == ^[width] / 2;
+      height: == ^[height];
+      top: == :next[top];
+      right: == :next[left];
+    }
+  }
 }
 
-/* Not possible with Media Queries */
 {% endhighlight %}
 
+When the `section` element `width` goes bellows 400px, the `article` elements will be constrain
+by the `@if` block otherwise by the `@else` block.
+
+### Constraints evaluation in conditions
+In order to determine if some conditions hold, GSS needs to evaluate every constraints outside of those conditions before evaluating if
+they are met, which include constraints appearing after the conditions declaration.
```

Comment:
I find it easier to think about if it's phrased like this:

"In order to determine if a condition holds, GSS needs to evaluate every constraint outside of that conditional before evaluating if it is met. This includes constraints appearing after the conditional declaration."